code/hud/hudtarget.cpp

```diff
@@ -5533,6 +5533,16 @@ void HudGaugeWeaponEnergy::initBitmaps(char *fname)
 	Energy_bar.first_frame = bm_load_animation(fname, &Energy_bar.num_frames);
 	if ( Energy_bar.first_frame < 0 ) {
 		Warning(LOCATION,"Cannot load hud ani: %s\n", fname);
+	} else {
```

Comment:
According to git diff -w, that else block is indeed the only non-whitespace change.
For anyone's reference who may be curious, the easiest way to get that info is:
```
# Fetch the pr to a local reference, branch pr1270
git fetch origin pull/1270/HEAD:pr1270
 # Diff (or --stat diff) of the branch to its common ancestor with master, ignoring whitespace (-w)
git diff -w [--stat] `git merge-base origin/master pr1270` pr1270
```